code/modules/mob/living/silicon/silicon.dm

```diff
@@ -63,7 +63,7 @@
 
 /mob/living/silicon/stun_effect_act(var/stun_amount, var/agony_amount)
 	return	//immune
-
+/*
```

Comment:
Feel free to purge this entirely rather than commenting out, I'll be re-doing silicons as a whole anyway very shortly